.gitignore

```
@@ -1,4 +1,5 @@
 .DS_Store
 .idea
 /node_modules
-/dist
\ No newline at end of file
+/dist
+/src/.vscode
```

Можно лучше: убрать лишнее ограничение при игнорировании директории.
Выражение `/src/.vscode` означает, что гит будет игнорировать все директории `.vscode` внутри директории `/src`. Но может быть ситуация, когда `.vscode` будет создана внутри корневой директории. Гит будет отображать в списке `untracked files`, которые по ошибке можно будет случайно добавить и запушить. Чтобы этого избежать, лучше всего игнорировать `.vscode` независимо от вложенности:
 ```suggestion
.vscode
```